src/Abp/WebHooks/IWebHooksConfiguration.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Abp.Collections;
+using Abp.Json;
+using Newtonsoft.Json;
+
+namespace Abp.WebHooks
+{
+    public interface IWebHooksConfiguration
+    {
+        /// <summary>
+        /// HttpClient timeout. Sender will wait <c>WebHookTimeout</c> second before throw timeout exception
+        /// </summary>
+        TimeSpan WebHookTimeout { get; set; }
```

Comment:
how about `TimeoutDuration` without `Webhook` prefix since this property is already in `Webhook` configuration